src/components/picker/PickerItem.tsx

```diff
@@ -65,16 +64,12 @@ const PickerItem = (props: PickerItemProps) => {
   const _onPress = useCallback(async (props: any) => {
     // Using !(await onPress?.(item)) does not work properly when onPress is not sent
     // We have to explicitly state `false` so a synchronous void (undefined) will still work as expected
-    if (onPress && await onPress(context.isMultiMode ? !isSelected : undefined, props) === false) {
+    if (onPress && (await onPress(context.isMultiMode ? !isSelected : undefined, props)) === false) {
```

Comment:
Did you add it manually or is it prettify?

Reply:
prettify